docs/cli/usage/customcf.md

```diff
@@ -1,18 +1,24 @@
 ---
-title: Custom AWS Resources
+title: Add custom AWS resources
 description: The Amplify CLI provides escape hatches for modifying the backend configurations generated in the form of CloudFormation templates by the CLI. This allows you to use the CLI for common flows but also any advanced scenarios which aren't provided in the standard category workflows.
 ---
 
-The Amplify CLI provides escape hatches for modifying the backend configurations generated in the form of CloudFormation templates by the CLI. This allows you to use the CLI for common flows but also any advanced scenarios which aren't provided in the standard category workflows.
+The Amplify CLI provides escape hatches for modifying the backend configurations generated in the form of CloudFormation templates by the CLI. 
 
-* For your GraphQL API generated by the Amplify CLI, you have the ability to [override resolvers](~/cli/graphql-transformer/resolvers.md#overwriting-resolvers) as well as add your own [custom resolvers](~/cli/graphql-transformer/resolvers.md#custom-resolvers) to add on to the CLI generated CloudFormation stacks.
+## How to extend Amplify CLI
 
-* For majority of the other categories, you can locally edit and manage the CloudFormation file generated by the CLI in the `amplify/backend/<category>/<cloudformation-template.json/yml>` location. [Read more about Cloudformation](https://aws.amazon.com/cloudformation/)
+The most common scenarios for extending your app beyond Amplify CLI's basic features are to:
+* customize your GraphQL API business logic by [overriding resolvers](~/cli/graphql-transformer/resolvers.md#overwriting-resolvers) or author your own [custom resolvers](~/cli/graphql-transformer/resolvers.md#custom-resolvers).
+* override Amplify CLI's generated CloudFormation by editing `amplify/backend/<category>/<cloudformation-template.json/yml>`. [Read more about Cloudformation](https://aws.amazon.com/cloudformation/)
+* add custom AWS resources that are not supported by the Amplify CLI currently, follow the steps below.
 
-* For adding and deploying a CloudFormation stack for an AWS Service not supported by the Amplify CLI currently, go through the following steps:
+## Add custom AWS resources 
 
-1. Modify `amplify/backend/backend-config.json` in your project:
-```
+Leverage CloudFormation to add an AWS resource to your Amplify project that isn't supported out-of-the-box.
+
+### 1. Configure backend configuration with custom resource
```

Comment:
This seems to jump right into it. Should we explain "Amplify is partitioned into Categories of use cases like 'Auth' or 'Functions'. To add a custom stack you will create a category like 'Custom'...."

Reply:
Done.